### PR TITLE
Test code style example files on CI

### PR DESCRIPTION
**Pull request by martijnversluis**
This is the first step setting up CI for Launch Base. It only runs Rubocop, Reek and ESlint against the example files using the config files. This enables a somewhat test driven approach on adjusting the configurations to our needs.

### CODE REVIEW

File: launch_base.gemspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+require "launch_base/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "launch_base"
+  spec.version       = LaunchBase::VERSION
+  spec.authors       = ["Martijn Versluis"]
+  spec.email         = ["martijn@kabisa.nl"]
+
+  spec.summary       = %q{A Rails template with Kabisa defaults}
```
Comment (drumusician) [inline launch_base.gemspec]: why suddenly the %q ?

File: launch_base.gemspec [hunk 2]
```diff
@@ -0,0 +1,35 @@
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require "launch_base/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "launch_base"
+  spec.version       = LaunchBase::VERSION
+  spec.authors       = ["Martijn Versluis"]
+  spec.email         = ["martijn@kabisa.nl"]
```
Comment (drumusician) [inline launch_base.gemspec]: if there is only 1 email, maybe use 'martijn@kabisa.nl'

File: launch_base.gemspec [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise "RubyGems 2.0 or newer is required to protect against " \
+      "public gem pushes."
+  end
+
+  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = "exe"
```
Comment (drumusician) [inline launch_base.gemspec]: is there an 'exe' directory? The convention is bin, right?

**Comment by martijnversluis** [comment]
Closing in an attempt to get correct CodeClimate feedback.

**Pull request closed without merging**